waku/store/v3/store.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+// 13/WAKU2-STORE rfc: https://rfc.vac.dev/spec/13/
+// Protocol identifier: /vac/waku/store/3.0.0
+package waku.store.v3;
+
+import "waku/message/v1/message.proto";
+
+
+message HistoryRequest {
+  string request_id = 1;
+  optional string pubsub_topic = 10;
+  repeated string content_topic = 11;
+  optional int64 time_start = 12;
```

Comment:
Should this be a `uint` as we don't expect queries before 1970?

Reply:
This is a good point. 

For consistency on how time types are specified outside of Waku (e.g., the Linux kernel defines its time types as `int64`), let's keep the signed version of the timestamp.

In any case, the `int64` is an efficient yet future-proof type for our purposes ([from proto3 docs](https://developers.google.com/protocol-buffers/docs/proto3#:~:text=int-,int64,-Uses%20variable%2Dlength)):

> **int64** | Uses variable-length encoding. Inefficient for encoding negative numbers – if your field is likely to have negative values, use `sint64` instead.